index.php

```diff
@@ -1,26 +1,24 @@
-<?php /*
+<?php
+/*
```

Comment:
I know it's different in other files, too, but as you've made a change here, can you make it like it is in `api_db.php` please? 

https://github.com/pi-hole/AdminLTE/blob/master/api_db.php#L2

```
/*   Pi-hole: A black hole for Internet advertisements
```

Reply:
Done